hack/common.sh

```diff
@@ -48,8 +48,8 @@ NOOBAA_CSV="$OUTDIR_TEMPLATES/noobaa-csv.yaml"
 ROOK_CSV="$OUTDIR_TEMPLATES/rook-csv.yaml.in"
 OCS_CSV="$OUTDIR_TEMPLATES/ocs-operator.csv.yaml.in"
 
-LATEST_ROOK_IMAGE="rook/ceph:v1.10.0"
-LATEST_NOOBAA_IMAGE="noobaa/noobaa-operator:master-20220916"
+LATEST_ROOK_IMAGE="docker.io/rook/ceph:v1.10.0"
+LATEST_NOOBAA_IMAGE="quay.io/noobaa/noobaa-operator:master-20220916"
```

Comment:
Can we use these variables to construct URL for noobaa and rook like we do for bundle?

https://github.com/red-hat-storage/ocs-operator/blob/316e2b3235bfaf0878a949ad3155c807f58791ec/hack/common.sh#L92

Reply:
ACK, done.